qemu/tests/cfg/multi_disk.cfg

```diff
@@ -106,10 +106,9 @@
                     stg_params = "drive_format:scsi-block "
                     kill_vm = yes
                     force_create_image = no
-                    pre_command = "modprobe -r scsi_debug; modprobe sg; modprobe scsi_debug add_host=9 dev_size_mb=50"
                     post_command = "rmmod scsi_debug"
-                    disk_check_cmd = "ls -1d /dev/nvme* || ls -1d /dev/sda*"
-                    image_stg_blacklist = "/dev/sda[\d]* /dev/sg0"
+                    disk_create_cmd = "modprobe -r scsi_debug; modprobe sg; modprobe scsi_debug add_host=9 dev_size_mb=50"
+                    disk_check_cmd = "ls /dev/s[dg]* -1A"
```

Comment:
suggest the logic simpler 
disk_check_cmd = "ls /dev/s[dg]* -1A
pre_command = ${disk_check_cmd} > /tmp/old_disk &&
pre_command += "modprobe -r scsi_debug; modprobe sg; modprobe scsi_debug add_host=9 dev_size_mb=50 &&"
pre_command += ${disk_check_cmd} > /tmp/new_disk && comm -3 /tmp/old_disk  /tmp/new_disk  > /tmp/add_disk 


then your py become simpler also: